### PR TITLE
Navigation Fix and Style changes

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**
Fixed navigation error on setup complete. Fixed style on livery sources.

### CODE REVIEW

File: src/components/styles/baseline.less [hunk 1]
```diff
@@ -32,3 +32,15 @@ html {
   right: 0;
   bottom: 0;
 }
+
```
Comment (davwheat) [inline src/components/styles/baseline.less:35]: This *will not work* in prod.

You need to use the `makeStyles` and `useStyles` helpers from Material UI.

Very brief example is here: https://github.com/MSFS-Mega-Pack/MSFS2020-livery-manager/blob/a93d9188d1e17a5a2126b7b6233833029b36fdb0/src/pages/SplashScreen.js#L9

**Review by davwheat — CHANGES_REQUESTED** [review]

File: src/components/styles/baseline.less (hunk 1 above)
Comment (NinjaLabs-Dev) [reply src/components/styles/baseline.less:35]: How would we target the table header and last row though?

**Comment by NinjaLabs-Dev** [comment]
Updated using makeStyles instead 😄

**Review by davwheat — APPROVED** [review]
Looks good! Should get this merged AFTER #9

**Comment by GewoonJaap** [comment]
> Looks good! Should get this merged AFTER #9

Maybe we can rebase on the #9 branche? And then merge?

**Comment by davwheat** [comment]
Good idea. I'll do that.

@GewoonJaap  Could you review #9? See if there are any glaring problems or improvements I should make before the rebase.

**Comment by davwheat** [comment]
Rebase complete

**Pull request merged**